gammapy/modeling/models/temporal.py

```diff
@@ -587,7 +596,10 @@ def write(self, filename, format="table", overwrite=False):
             table = self.to_table()
             table.write(filename, overwrite=overwrite)
         elif format == "map":
-            self.map.write(filename, overwrite=overwrite)
+            # RegionNDMap.from_hdulist does not update the header
+            hdulist = self.map.to_hdulist()
+            hdulist[0].header.update(time_ref_to_dict(self.tref_mjd))
```

Comment:
Same comment here. Don't export the time reference to the primary header. Use the time axis hdu instead.